src/room.ts

```diff
@@ -10,16 +10,17 @@ import { Message }    from './message'
 import { UtilLib }    from './util-lib'
 
 type RoomObj = {
-  id:         string,
-  encryId:    string,
-  topic:      string,
-  ownerUin:   number,
-  memberList: Contact[],
-  nameMap:    Map<string, string>,
-  aliasMap:   Map<string, string>,
+  id:               string,
+  encryId:          string,
+  topic:            string,
+  ownerUin:         number,
+  memberList:       Contact[],
+  nameMap:          Map<string, string>,
+  aliasMap:     Map<string, string>,
+  contactAliasMap:  Map<string, string>,
 }
 
-type NameType = 'name' | 'alias'
+type NameType = 'name' | 'alias' | 'contactAlias'
```

Comment:
What I had suggested is to use `alias` as an alias of `roomAlias`, which means here should be:

`type NameType = 'name' | 'alias' | 'roomAlias' | 'contactAlias'`

What you should do is to use the `roomAlias` and `contactAlias` in document, and treat `alias` an alias of `roomAlias`.